assignments/this.js

```diff
@@ -1,26 +1,54 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window/Global Object Binding - if "this" is used while in global scope, it will bind to the window object;
+* 2. Implicit Binding -used when referring to a property within an object, the word this followed by .
+* 3. New binding - used when creating an object constructor. When the constructor is used to create a specific instance 
+the format of a new binding tells the parameters to attach in the new object being created from the constructor
```

Comment:
Yes, the new keyword creates an empty object in the constructor function and sets `this` to reference that object